spec/features/courses/start_date/edit_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+feature 'Edit course start date', type: :feature do
+  let(:current_recruitment_cycle) { build(:recruitment_cycle) }
+  let(:start_date_page) { PageObjects::Page::Organisations::CourseStartDate.new }
+  let(:course_details_page) { PageObjects::Page::Organisations::CourseDetails.new }
+  let(:provider) { build(:provider) }
+
+  before do
+    stub_omniauth
+    stub_api_v2_request(
+      "/recruitment_cycles/#{current_recruitment_cycle.year}",
+      current_recruitment_cycle.to_jsonapi
+    )
+    stub_api_v2_request(
+      "/recruitment_cycles/#{current_recruitment_cycle.year}" \
+      "/providers/#{provider.provider_code}?include=courses.accrediting_provider",
+      build(:provider).to_jsonapi(include: %i[courses accrediting_provider])
+    )
+
+    stub_course_request
+    stub_course_details_tab
+    start_date_page.load_with_course(course)
+  end
+
+  context 'a course with a start date of october 2019' do
+    let(:course) do
+      build(
+        :course,
+        start_date: 'October 2019',
+        edit_options: {
+          start_dates: ['October 2019', 'November 2019']
+        },
+        provider: provider
+      )
+    end
+
+    scenario 'can cancel changes' do
+      click_on 'Cancel changes'
+      expect(course_details_page).to be_displayed
+    end
+
+    xscenario 'can navigate to the edit screen and back again' do
```

Comment:
Oh, we need to create a separate card to wire this functionality in and then turn this test on.

Reply:
Created, https://trello.com/c/47CUNq1S/1940-add-link-to-edit-course-start